Bio/SeqFeature.py

```diff
@@ -335,6 +335,72 @@ def extract(self, parent_sequence):
                              "sequence file for a valid location.")
         return self.location.extract(parent_sequence)
 
+    def translate(self, parent_sequence, table="Standard", stop_symbol="*",
```

Comment:
Note on #1103/#1144 for the ``SeqRecord`` translate method, I put the arguments from the ``Seq`` objetct's ``.translate(...)`` method before those specific to the record annotation - but at the time wavered on this ordering.

I see you've put the ``parent_sequence`` first, followed by all the arguments from the ``Seq`` objetct's ``.translate(...)`` method. That seems fairly logical, but see also #808 - we might add another argument to the ``.extract(...)`` method later with a dictionary of referenced source sequences.